modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -199,7 +199,7 @@ rpk security acl delete ... --allow-role <role_name>
 rpk security acl delete ... --deny-role <role_name>
 ----
 
-When using the xref:reference:rpk/rpk-security/rpk-security-acl-delete[`rpk security acl delete`] command, Redpanda deletes all ACLs matching the parameters supplied. Be careful to closely match the exact ACL you wish to delete when using this command. If you supply only the `--allow-role` parameter, for example, Redpanda will delete every ACL granting that role authorization to a resource.
+When using the `rpk security acl delete` command, Redpanda deletes all ACLs matching the parameters supplied. Be careful to closely match the exact ACL you wish to delete when using this command. If you supply only the `--allow-role` parameter, for example, Redpanda will delete every ACL granting that role authorization to a resource.
```

Comment:
```suggestion
When you execute the `rpk security acl delete` command, Redpanda deletes all ACLs matching the parameters supplied. Make sure to match the exact ACL you want to delete. If you supply only the `--allow-role` parameter, for example, Redpanda will delete every ACL granting that role authorization to a resource.
```

Reply:
removed the reference link for the command?

Reply:
I removed it because just below the paragraph there are links to all three commands.